### PR TITLE
PR for project review

### PR DESCRIPTION
**Pull request by leonmezu1**
# The following requirements were met during the development of this project:

 -   Please make different requests for the Celsius data and the Fahrenheit data.
 -   Please remember about error handling for promises or async/await.
 -   Use webpack introduced in previous lesson.
 -       Remember to include all webpack related packages in devDependencies.
 -       Remember to add scripts which are necessary to build your project in package.json.
 -     Remember to add node_modules dir to .gitignore.
 -  Use ES6 introduced in previous lesson.

### **_An OpenWeather API KEY might be necessary to review this project, feel free to reach me if you need it._**

### CODE REVIEW

File: src/js/modules/index.old.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/* /* eslint-disable func-names
```
Comment (mcrd25) [inline src/js/modules/index.old.js]: You ought to remove commented out code:
It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history 

see this [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)

Please remove

**Review by mcrd25 — APPROVED** [review]
# Changes Required :mag: 

Hi @leonmezu1 :wave: 

Excellent effort! :clap: 
Only minor changes required. See my review comments.


***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: src/js/modules/index.old.js (hunk 1 above)
Comment (leonmezu1) [reply src/js/modules/index.old.js]: Deleted

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Thank you for applying the changes. You can merge it :+1: 
Good continuation :handshake: 

Thanks
Mael FOSSO

**Pull request merged**